tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3002,7 +3043,7 @@ def ssim(img1, img2, max_val):
 
 
 @tf_export('image.ssim_multiscale')
-def ssim_multiscale(img1, img2, max_val, power_factors=_MSSSIM_WEIGHTS):
+def ssim_multiscale(img1, img2, max_val, power_factors=_MSSSIM_WEIGHTS, **kwargs):
```

Comment:
Same thing here, explicitly name the arguments and document them

Reply:
should i go with kwargs (as no one changes them most or the times) or explicitly name the arguments?

Reply:
Explicitly name the arguments, please